YoutubeDownloader.Core/Downloading/VideoDownloadOption.cs

```diff
@@ -62,7 +62,7 @@ IEnumerable<VideoDownloadOption> GetVideoAndAudioOptions()
                         : [];
 
                     // If there are language-specific streams, include them all
-                    if (languageSpecificAudioStreamInfos.Any())
+                    if (languageSpecificAudioStreamInfos.Length != 0)
```

Comment:
```suggestion
                    if (languageSpecificAudioStreamInfos.Any())
```

Keep `Any()`, it's more consistent